lib/app/widgets/error_widgets.dart

```diff
@@ -28,55 +28,89 @@ void _showStackTrace(
   ));
 }
 
-class _MessageAndActions {
-  _MessageAndActions(this.message, this.actions);
+class PinkStripedErrorWidget extends StatelessWidget {
+  const PinkStripedErrorWidget(this.error, this.stackTrace);
 
-  factory _MessageAndActions.of(
-      BuildContext context, dynamic error, StackTrace stackTrace) {
-    String message;
-    final actions = <Widget>[];
-    final s = context.s;
+  final dynamic error;
+  final StackTrace stackTrace;
 
-    if (error is NoConnectionToServerError) {
-      message = s.app_errorScreen_noConnection;
-    } else if (error is AuthenticationError) {
-      message = s.app_errorScreen_authError;
-      actions.add(SecondaryButton(
-        onPressed: () => signOut(context),
-        child: Text(s.general_signOut),
-      ));
-    } else {
-      message = s.app_errorScreen_unknown(exceptionMessage(error));
-      actions.add(SecondaryButton(
-        onPressed: () => _showStackTrace(context, error, stackTrace),
-        child: Text(s.app_errorScreen_unknown_showStackTrace),
-      ));
-    }
+  @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(builder: (context, constraints) {
+      final size = constraints.biggest;
+      final diagonal = sqrt(pow(size.width, 2) + pow(size.height, 2));
+      final numSegments = diagonal ~/ 50;
 
-    return _MessageAndActions(message, actions);
+      return Container(
+        decoration: BoxDecoration(
+          gradient: LinearGradient(
+            begin: Alignment.topLeft,
+            end: Alignment.bottomRight,
+            colors: [
+              for (var i = 0; i < numSegments; i++) ...[
+                Color(0xffe966aa),
+                Color(0xffe966aa),
+                Color(0xffff66aa),
+                Color(0xffff66aa),
+              ],
+            ],
+            stops: [
+              for (var i = 0; i < numSegments; i++) ...[
+                1.0 / numSegments * i,
+                1.0 / numSegments * (i + 0.5),
+                1.0 / numSegments * (i + 0.5),
+                1.0 / numSegments * (i + 1),
+              ],
+            ],
+          ),
+        ),
+        padding: const EdgeInsets.all(8),
+        child: GestureDetector(
+          onLongPress: () => _showStackTrace(context, error, stackTrace),
+          child: SafeArea(
+            child: Center(
+              child: Text(
+                '$error\nLong-tap to view stack trace.',
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  color: Colors.black,
+                  fontStyle: FontStyle.normal,
+                  fontSize: 16,
+                  fontFamily: 'Roboto',
+                  fontWeight: FontWeight.bold,
+                  decoration: TextDecoration.none,
+                ),
```

Comment:
The thing is, this is a pretty special text style. I don't think there's a small bold text style.
And the `fontStyle` and `decoration` setters are necessary because this widget might be displayed outside of a `Scaffold` where the default text style is quite ugly.